src/msg/async/AsyncMessenger.cc

```diff
@@ -167,7 +167,7 @@ int Processor::rebind(const set<int>& avoid_ports)
   ldout(msgr->cct, 10) << __func__ << " new nonce " << nonce << " and inst " << msgr->my_inst << dendl;
 
   ldout(msgr->cct, 10) << __func__ << " will try " << addr << " and avoid ports " << new_avoid << dendl;
-  return bind(addr, new_avoid);
+  return bind(msgr->my_inst.addr, new_avoid);
```

Comment:
@tchaikov  @yuyuyu101 
Oke, good. I'm testing that now.
I need a godd IRC client that does not disconnect every 5 minutes, AND where I can kill all un/subscribe messages. Otherwise I just lose track of the discussions.
Challenge is that my terminals run W10.